test/cli.ava.js

```diff
@@ -0,0 +1,74 @@
+import Store from './store.js'
```

Comment:
Add `execa`.  Execute the real CLI once, passing in a real stdin.  This is slower than testing through the API, but will allow us to check that the linkage to the lib script is still working.